core/tests/protractor/accessibility.js

```diff
@@ -543,11 +543,13 @@ describe('screenreader and keyboard user accessibility features', function() {
   it('should move focus to skip to main content button in exploration player',
     async function() {
       // Should create and play a dummy exploration.
+      // expectWelcomeModal: true.
```

Comment:
I don't think you need comments like this. In fact, they could be confusing to someone who isn't familiar with the `createAndPublishTwoCardExploration` function. Ditto elsewhere

Reply:
Done !

Reply:
I still see some comments like this. See my review below